perllib/FixMyStreet/App/Controller/Admin.pm

```diff
@@ -1470,6 +1479,11 @@ sub user_edit : Path('user_edit') : Args(1) {
         $c->forward('user_hide_everywhere', [ $user ]);
     } elsif ( $c->get_param('submit') and $c->get_param('remove_account') ) {
         $c->forward('user_remove_account', [ $user ]);
+    } elsif ( $c->get_param('submit') and $c->get_param('send_login_email') ) {
+        my $email = lc $c->get_param('email');
+        my @args = ( $email );
+        push @args, ( '', $id ) if $user->email ne $email || !$user->email_verified;
```

Comment:
Not really a big issue, but the empty string here makes me thing you should be passing a hash of email/id here (and email/name in the other place), rather than a list.